docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -2323,6 +2323,7 @@ This section contains reference information about each of the configuration prop
 |`fhirServer/security/oauth/smart/enabled`|boolean|Whether or not the server is enabled for OAuth-based authentication/authorization|
 |`fhirServer/security/oauth/smart/scopes`|array|The list of SMART scopes to advertise in the `.well-known/smart-configuration endpoint|
 |`fhirServer/security/oauth/smart/capabilities`|array|The list of SMART capabilities to advertise in the `.well-known/smart-configuration endpoint|
+|`fhirServer/security/validateSecurityContext`|boolean| Whether or not the securityContext validation is enabled for Binary resources|
```

Comment:
I wonder if we should put the word "Binary" into the config property if its only used for Binary resources.
```suggestion
|`fhirServer/security/validateBinarySecurityContext`|boolean| Whether or not the securityContext validation is enabled for Binary resources|
```
we'd need to update it in all the places, but I do think this is better...